mypy/test/testcmdline.py

```diff
@@ -85,7 +89,7 @@ def test_python_cmdline(testcase: DataDrivenTestCase, step: int) -> None:
     env.pop("COLUMNS", None)
     extra_path = os.path.join(os.path.abspath(test_temp_dir), "pypath")
     env["PYTHONPATH"] = PREFIX
-    env["__MYPY_UNDER_TEST__"] = "1"
+    env["__MYPY_UNDER_TEST__"] = "1" if based else "2"
```

Comment:
what do these numbers mean

Reply:
'1' means test mode, '2' means upstream tests mode (legacy), I've added a comment about it